unit_tests/src/test_rc_update_replica_access_time.cpp

```diff
@@ -50,18 +50,222 @@ TEST_CASE("#8260: rc_update_replica_access_time returns error on invalid inputs"
+    {
+        // clang-format off
+        const auto input = json{
+            {"access_time_updates", json::array_t{
+                {
+                    {"data_id", 0},
+                    {"atime", "00000000000"}
+                }
+            }}
+        }.dump();
+        // clang-format on
+
+        irods::experimental::client_connection conn;
+        char* output{};
+        CHECK(rc_update_replica_access_time(static_cast<RcComm*>(conn), input.c_str(), &output) == SYS_LIBRARY_ERROR);
+    }
+
+    SECTION("missing atime property")
+    {
+        // clang-format off
+        const auto input = json{
+            {"access_time_updates", json::array_t{
+                {
+                    {"data_id", 0},
+                    {"replica_number", 0}
+                }
+            }}
+        }.dump();
+        // clang-format on
+
+        irods::experimental::client_connection conn;
+        char* output{};
+        CHECK(rc_update_replica_access_time(static_cast<RcComm*>(conn), input.c_str(), &output) == SYS_LIBRARY_ERROR);
+    }
+
+    SECTION("null value for data_id")
+    {
+        // clang-format off
+        const auto input = json{
+            {"access_time_updates", json::array_t{
+                {
+                    {"data_id", nullptr},
+                    {"replica_number", 0},
+                    {"atime", "00000000000"}
+                }
+            }}
+        }.dump();
+        // clang-format on
+
+        irods::experimental::client_connection conn;
+        char* output{};
+        CHECK(rc_update_replica_access_time(static_cast<RcComm*>(conn), input.c_str(), &output) == SYS_LIBRARY_ERROR);
+    }
+
+    SECTION("null value for replica_number")
+    {
+        // clang-format off
+        const auto input = json{
+            {"access_time_updates", json::array_t{
+                {
+                    {"data_id", 0},
+                    {"replica_number", nullptr},
+                    {"atime", "00000000000"}
+                }
+            }}
+        }.dump();
+        // clang-format on
+
+        irods::experimental::client_connection conn;
+        char* output{};
+        CHECK(rc_update_replica_access_time(static_cast<RcComm*>(conn), input.c_str(), &output) == SYS_LIBRARY_ERROR);
+    }
+
+    SECTION("null value for atime")
+    {
+        // clang-format off
+        const auto input = json{
+            {"access_time_updates", json::array_t{
+                {
+                    {"data_id", 0},
+                    {"replica_number", 0},
+                    {"atime", nullptr}
+                }
+            }}
+        }.dump();
+        // clang-format on
+
+        irods::experimental::client_connection conn;
+        char* output{};
+        CHECK(rc_update_replica_access_time(static_cast<RcComm*>(conn), input.c_str(), &output) == SYS_LIBRARY_ERROR);
+    }
+}
+
+TEST_CASE("#8260: rc_update_replica_access_time does not return an error on bad atime values")
```

Comment:
I think this label is fine, but it's actually more than not returning an error: It's actually expected that the value is successfully updated to an invalid value if one is given. Maybe it could be...

```suggestion
TEST_CASE("#8260: rc_update_replica_access_time updates atime with bad atime values")
```

Reply:
Sure, that works. Will update.

Reply:
Done.